laravel-app/app/Services/PostService.php

```diff
@@ -10,7 +10,6 @@ class PostService implements PostServiceInterface
 {
     /**
      * To create a new post.
-     * @param User $user
      * @param array<mixed, mixed> $validatedData
```

Comment:
removee!!!!

Reply:
Already removed sir, as marked red by the git. Thanks 🚀